### PR TITLE
Add MockResponse with reasons

### PR DESCRIPTION
**Pull request by pkarman**
**Why**: Better mocking of raw vendor response.

### CODE REVIEW

File: lib/proofer/vendor/mock_response.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+require 'hashie'
+
+module Proofer
+  module Vendor
+    class MockResponse < Hashie::Mash
```
Comment (zachmargolis) [inline lib/proofer/vendor/mock_response.rb]: instead of an entire dependence on Hashie, what about just using an OpenStruct here?

Comment (pkarman) [reply lib/proofer/vendor/mock_response.rb]: fair enough. I will try and create a new habit.

**Review by zachmargolis — APPROVED** [review]
LGTM, thanks!

File: lib/proofer/vendor/mock_response.rb (hunk 1 above)
Comment (monfresh) [reply lib/proofer/vendor/mock_response.rb]: I would go a step further and use a PORO. OpenStructs are slow and less flexible. http://www.schneems.com/2014/12/15/hashie-considered-harmful.html

**Pull request merged**